programs/uperf/run

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# - nr_threads
+# - protocol
+# - winsize
+# - runtime
+# - datasize
+
+run_uperf()
+{
+	local uperf_path
+	if [ -f /usr/bin/uperf ]; then
+		uperf_path=/usr/bin/uperf
+	elif [ -f /lkp/benchmarks/uperf/usr/local/bin/uperf ]; then
+		uperf_path=/lkp/benchmarks/uperf/usr/local/bin/uperf
```

Comment:
is it possible to favor this first? Or to make sure we always use benchmark one, we can only consider this situation, to do "cd lkp/benchmarks/uperf" like other test suite?